cityofaustin/atd-moped
# Removes deleted contracts from project list

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10342

## Testing
**URL to test:** 

**Steps to test:**
create two new contractors for a project, see them in list view. delete a contract, it should not display in list view.

---
#### Ship list
